### PR TITLE
Draft the Ember Data 2.3 release blog post

### PR DESCRIPTION
**Pull request by bmac**
cc @pangratz @fivetanley for review

### CODE REVIEW

File: source/blog/2016-01-11-ember-data-2-3-released.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+title: Ember Data 2.3 and 2.4 Beta Released
+author: Brendan McLoughlin
+tags: Releases
+---
+
+Ember Data 2.3, a minor version release of Ember Data, is released today. This release represents the work of over 32 direct contributors, and over 191 commits.
```
Comment (bmac) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: The 2.3 release hasn't been cut yet so these numbers may change.

File: source/blog/2016-01-11-ember-data-2-3-released.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+For more information you can check out Ember's
+[feature flags guide](https://guides.emberjs.com/v2.2.0/configuring-ember/feature-flags/).
+
+#### `ds-find-include`
+
+The `ds-find-include` feature allows an `include` query parameter to
+be specified with using `store.findRecord()` and `store.findAll()` as
+described in [RFC 99](https://github.com/emberjs/rfcs/pull/99). This
+should make it easier to specify when backends should return
+sideloaded relationships.
```
Comment (fivetanley) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: Would be great to shoutout @HeroicEric for this awesome work!

File: source/blog/2016-01-11-ember-data-2-3-released.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+var article = this.store.findAll('article', { include: 'comments' });
+```
+
+#### `ds-references`
+
+The `ds-references` feature implementes the refernces api as described
+in [RFC 57](https://github.com/emberjs/rfcs/pull/57). References is a
+low level API to perform meta-operations on records, has-many
+relationships and belongs-to relationships:
```
Comment (fivetanley) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: Would be great to thank @pangratz for his work!

File: source/blog/2016-01-11-ember-data-2-3-released.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+### Changes in Ember Data 2.3
+
+Ember Data 2.3 is now a full Ember CLI addon. Previous versions of
+Ember Data requried users to include both the *ember-data* package in
+their project's `package.json` file and the *ember-data* package in
+their project's `bower.json` file. This resulted in confusion because
+updating *ember-data* in `package.json` would not update *ember-data*
+in `bower.json`.
+
+To upgrage Ember Data to version 2.3 in your Ember CLI projects please
```
Comment (acorncom) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: *upgrade*

File: source/blog/2016-01-11-ember-data-2-3-released.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+---
+
+Ember Data 2.3, a minor version release of Ember Data, is released today. This release represents the work of over 32 direct contributors, and over 191 commits.
+
+Ember Data 2.4 beta.1, the branch of Ember Data that will be released as stable in roughly six weeks, is also being released today.
+
+### Changes in Ember Data 2.3
+
+Ember Data 2.3 is now a full Ember CLI addon. Previous versions of
+Ember Data requried users to include both the *ember-data* package in
```
Comment (HeroicEric) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: "required"

File: source/blog/2016-01-11-ember-data-2-3-released.md (hunk 4 above)
Comment (fivetanley) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: I think this is intentional. I only upgrade my software in states of rage. :P

Comment (acorncom) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: Oh, in that case, suggestion retracted ... :trollface:

File: source/blog/2016-01-11-ember-data-2-3-released.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+      'ds-references': true
+    }
+  }
+};
+```
+
+For more information you can check out Ember's
+[feature flags guide](https://guides.emberjs.com/v2.2.0/configuring-ember/feature-flags/).
+
+#### `ds-find-include`
```
Comment (pangratz) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: This feature has been implemented as `ds-finder-include`, but it is named `ds-find-include` in `FEATURES.md`. I've opened emberjs/data#4059 to clarify this and will report back...

**Comment by pangratz** [comment]
LGTM. I've opened bmac/website#1 with typo fixes and added code samples for the references feature.

File: source/blog/2016-01-11-ember-data-2-3-released.md (hunk 6 above)
Comment (pangratz) [inline source/blog/2016-01-11-ember-data-2-3-released.md]: The feature is `ds-finder-include`, as pointed out by @HeroicEric https://github.com/emberjs/data/pull/4059#issuecomment-170882397. My PR linked below updates this...

**Pull request merged**